schemup/errors.py

```diff
@@ -1,5 +1,6 @@
 class ValidationMismatch(object):
-    def __init__(self, (tableName, actual, expected)):
+    def __init__(self, xxx_todo_changeme):
+        (tableName, actual, expected) = xxx_todo_changeme
```

Comment:
Can we call it `mismatch` ?
*Mismatch classes were instantiated from `schemup/commands.py:validate()`, and the variables there to bootstrap such classes were called `mismatches`.